components/brave_webtorrent/extension/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "version": "1.0.0",
+  "name": "Brave Webtorrent",
+  "manifest_version": 2,
+  "description": "Webtorrent extension",
+  "background" : {
+    "scripts": ["extension/out/brave_webtorrent_background.bundle.js"],
+    "persistent": true
+  },
+  "permissions": ["tabs", "windows"],
+  "sockets": {
+    "udp": {
+      "send": "*",
+      "bind": "*"
+    },
+    "tcp": {
+      "connect": "*"
+    },
+    "tcpServer": {
+      "listen": "*:*"
+    }
+  },
+  "key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEArfOx1MW/cb3YPNlmT37CuISYgRbtR1SIdgnx/cfTyXO/PuD1VVsQWLDmrZGDmYVCzZvP36t75uhpJH4IoXL58U16yhdXZeSlb0LKcgMZB6cMNyjznV4NTEeY+tLnwGaB1TVdkJgSlY09psyfvcdzQd8xz9CNE6CXDzEq8+uMSaoAyEJ3nP78yV33nBrMj3jbjTi1fr2QsrpoISql/pJ9Zr5V0QbK4wIqln20ly96KuAO5c1DM9z9VnoYFdirEZBfkT/4gB7pBfyd4ScoMhXuaa9w53N8Espu1bC0RGmaKB679rGQdaBTrEUGF+PNfsucjnyrsnup6GMVhc91CXTDjQIDAQAB"
```

Comment:
would it need some CSP declared just for safety? cc @diracdeltas

Reply:
browser-laptop example: https://github.com/brave/browser-laptop/blob/master/app/extensions.js#L251

Reply:
Thanks for spotting this, I think I missed it, I'll use the setting from b-l here.